### DISCUSSION TITLE
Command Palette: selected item changes when list finishes loading

### DISCUSSION THREAD

**Discussion by buondevid** [community/community#17370] [Other Feature Feedback, Questions, & Ideas]
I like the feature but I'm experiencing a bothering problem: when I start writing in the Palette, usually a first item gets shown and usually, it's the one I'm interested in. The problem is that when I press Enter to select the item, the list finishes loading, and another element gets highlighted/selected, with the result that now I've entered the wrong repo/PR/whatever even tho I pressed Enter when the right one was highlighted.

**TLDR**: basically the list is swapping the selected item when it finishes loading, not very good UX-wise. Keep the first shown item always selected so that we can be confident about where we are ending up when pressing Enter fast.

**Comment by SebastiaanNijland** [comment]
This happens to me on a daily basis as well.
